pages/account-page.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: account-page
+template-title: Account
+template-description: Account landing page that displays orders, saved products, and personal information
+title: Account
+description: Account landing page that displays orders, saved products, and personal information
+cards:
+     - title: Orders
+       description: Track your recent orders and view your purchase history.
+     - title: Saved products
+       description: View or edit the products you have added to your list.
+account:
```

Comment:
This is a nice approach, but it results in two issues on the rendered page:
1. The emails and phone numbers can/should be linked.
2. Using paragraphs for each line results in awkward vertical spacing.

Issue 2 could be handled by either some utility classes on the paragraphs or looping with line breaks instead of paragraphs.

Issue 1 is more complicated. You can easily link them either in the markdown content area or the template. But it would make the looping more complicated since you would have to determine what it is. Maybe look at the "[case](https://shopify.github.io/liquid/tags/control-flow/)" command in liquid? I haven't used this much, but I think you would have to run this inside the for loop. Not sure. I'll leave it up to you.